Copy and adapt the release directory from coredev 5.2.

mauritsvanrees opened the pull request:
This contains a tox.ini and create-constraints script which helps in preparing a (pending) directory on dist.plone.org.
This replaces `scripts/create-constraints.py` and the `constraints` section from `core.cfg`.

This should be fine to merge, and should only be interesting for the release managers. But maybe someone is using the old `constraints` section. So here is your chance to speak up. :-)
I already did the same on 5.2 months earlier though.

I won't start tests on Jenkins, because this will have no influence. But the GH Actions should still pass.

jensens commented on release/create-constraints.py:
```diff
@@ -0,0 +1,46 @@
+# List packages that we should not include.
+# Originally this was for packages that are pinned by Zope
+# but cannot be pip installed on Py 2 or on Py 3.
+BLACKLIST = [
```
Well, better `DENYLIST`?

jensens replied:
```suggestion
DENYLIST = [
```

mauritsvanrees replied:
Ah, yes, same as in the hotfix. I will change it and then merge.

jensens commented on release/create-constraints.py:
```diff
@@ -0,0 +1,46 @@
+    cfile.write("# File created by {}\n".format(__file__))
+    cfile.write("# Constraints parsed from {}\n".format(config_file))
+    for package, version in sorted(versions.items()):
+        if package in BLACKLIST:
```
```suggestion
        if package in DENYLIST:
```

jensens reviewed (approved):
except for the tiny naming issue, LGTM.

mauritsvanrees committed:
Copy and adapt the release directory from coredev 5.2.

This contains a tox.ini and create-constraints script which helps in preparing a (pending) directory on dist.plone.org.
This replaces `scripts/create-constraints.py` and the `constraints` section from `core.cfg`.
